skyportal/handlers/api/source.py

```diff
@@ -241,6 +242,7 @@ async def get_source(
                     joinedload(FollowupRequest.allocation).joinedload(Allocation.group),
                     joinedload(FollowupRequest.requester),
                     joinedload(FollowupRequest.watchers),
+                    joinedload(FollowupRequest.transactions),
```

Comment:
You can specify the column too in the joinedload with `joinonly`. that way you don't query all the stuff you don't need (so you can get only the content that you use after).

Reply:
added.